src/js/RacingCar.js

```diff
@@ -0,0 +1,57 @@
+import { CAR_PROGRESS_THRESHOLD, CAR_NAME_RULE, TRY_COUNT_LIMIT } from './constants.js';
+
+export default class RacingCar {
+  cars;
+  tryCount;
+
+  constructor(cars = [], tryCount = 0) {
+    this.cars = cars;
+    this.tryCount = tryCount;
+  }
+
+  #isCarNameLengthValid(cars) {
+    return cars.every((name) => name.length >= CAR_NAME_RULE.MIN_LENGTH && name.length <= CAR_NAME_RULE.MAX_LENGTH);
+  }
+
+  #isCarNameDuplicated(cars) {
+    return new Set(cars).size !== cars.length;
+  }
+
+  registerCars(carNames) {
+    try {
+      const cars = carNames.split(',').map((name) => name.trim());
+
+      if (!this.#isCarNameLengthValid(cars)) throw new Error('자동차 이름은 1자 이상, 5자 이하여야 합니다.');
+      if (this.#isCarNameDuplicated(cars)) throw new Error('자동차 이름은 중복될 수 없습니다.');
```

Comment:
지금은 registerCars 내에서 throw를 발생시키면서 동시에 try/catch를 하고 있어요.

그보다는 이렇게
```suggestion
      this.#validateCarNameLength(cars);
      this.#validateDuplicatedCarName(cars);
```
각각의 메소드에서 throw를 해주면 어떨까요?